molgenis-bootstrap/src/main/java/org/molgenis/bootstrap/populate/UsersGroupsPopulatorImpl.java

```diff
@@ -70,12 +70,14 @@ public void populate()
 		anonymousUser.setSuperuser(false);
 		anonymousUser.setChangePassword(false);
 
-		// create all users group
-		Group allUsersGroup = groupFactory.create();
-		allUsersGroup.setName(AccountService.ALL_USER_GROUP);
+		// create user role
+		Role userRole = roleFactory.create();
+		userRole.setName(ROLE_USER);
+		userRole.setLabel("User");
```

Comment:
Should this render as "Gebruiker" when the active language is Dutch?

Reply:
Probably yes!